src/components/LoansByCategory/MFIRecommendations/MfiLoansWrapper.vue

```diff
@@ -18,7 +18,7 @@
 			<a
 				class="tw-text-action tw-text-base"
 				href="https://www.kiva.org/lend?partner=59&status=fundRaising&sortBy=newest"
-				v-kv-track-event="['Lending', 'click-MFI-feature', 'view all']"
+				v-kv-track-event="['event-tracking', 'click', 'mfi-view-all']"
```

Comment:
Is `event-tracking` what we want here? It seems too general. Maybe that's also part of the tracking ticket.

Reply:
we can fix it now, any suggestion from category list?

Reply:
lending is no more an options for categories, you can check available options here https://docs.google.com/spreadsheets/d/1pbxxYDfeOkrGhFi8HWl5K-vCPXaqO3bXS1Io3E_SDKc/edit#gid=1801894844

Reply:
Oh I see we are needing to use a new style here for event strings. If we don't have it already, please log a new ticket to work with PM/Data on the expected strings here. I'll resolve this comment for now.

Reply:
Yeah we'll need to expand the categories with the data team.